apps/mobile-wallet/app.config.js

```diff
@@ -37,7 +37,7 @@ export default {
       supportsTablet: true,
       bundleIdentifier: 'org.alephium.mobilewallet',
       infoPlist: {
-        BGTaskSchedulerPermittedIdentifiers: ['$(PRODUCT_BUNDLE_PACKAGE_TYPE)'],
+        BGTaskSchedulerPermittedIdentifiers: ['$(PRODUCT_BUNDLE_IDENTIFIER)'],
```

Comment:
This is mentioned in https://github.com/Rapsssito/react-native-background-actions/blob/master/INSTALL.md

<img width="1083" alt="image" src="https://github.com/user-attachments/assets/ecfc6554-6800-4a28-b2fe-cccc342765c2" />

Reply:
Ok, interesting